packages/plugin-packer/package.json

```
@@ -6,29 +6,31 @@
   "engines": {
     "node": ">=10"
   },
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "bin": {
     "kintone-plugin-packer": "bin/cli.js"
   },
   "files": [
     "bin",
-    "src",
+    "dist",
     "from-manifest.js"
   ],
   "scripts": {
-    "build": "run-p js css",
+    "build": "npm run build:ts && npm run build:site",
+    "build:site": "run-p js css",
     "lint": "npm run eslint",
```

The CI runs only the `test` and `lint` command. So it is necessary to add `tsc --noEmit` to check types and to include the command into the `lint` command.

```suggestion
    "lint": "run-p eslint lint:ts",
    "lint:ts": "tsc --noEmit",
```

---

`run-p lint:*` works fine! bc618d6